### PR TITLE
Add Makefile, tweak actions

### PR DESCRIPTION
**Pull request by silverwind**
- Add Makefile with dev tasks
- Clean up action files, remove unnecessary names
- Rename `deno` workflow to `ci`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,12 @@ deno run --allow-net --allow-env --allow-run --allow-sys src/webhook.ts
 This will spin up a web server on port 8000. You can then set up a GitHub
 webhook on `/trigger` to run this bot.
 
+## Development
+
+- `make fmt` runs the formatter
```
Comment (yardenshoham) [inline README.md:101]: currently it only checks formatting

Comment (silverwind) [reply README.md:101]: Ah, need to split

**Comment by silverwind** [comment]
Hmm hold on, I will include fmt-check in lint.

**Comment by yardenshoham** [comment]
Before merging, why make and not https://docs.deno.com/runtime/manual/tools/task_runner/?

**Comment by silverwind** [comment]
Ok should be good. All commands are quiet now which eliminates a lot of useless spam like module downloads or how many files were processed but keeps the important stuff. Only tidbit I have is that `fmt --check --quiet` does not show which files are unformatted with `--quiet` but it's acceptable as one can just run `make fmt`.

**Comment by silverwind** [comment]
Because I prefer Makefile in all my projects, e.g. I also don't use npm scripts because Makefile is programming-language agnostic and for certain tasks, much more flexible than these json scripts ever can be.

**Review by yardenshoham — APPROVED** [review]

**Comment by silverwind** [comment]
Also, `Makefile` nicely takes care of the globbing issue. In a deno script, one would likely have to detect the running shell because one can't just `shopt` in zsh for example.

**Pull request merged**